### PR TITLE
Modal dialog for a new document type

### PR DESCRIPTION
**Pull request by ComLock**

### CODE REVIEW

File: src/main/resources/assets/react/documentTypes/FieldsList.jsx [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+export const FieldsList = ({
+	collections,
+	globalFields,
+	interfaces,
+	servicesBaseUrl
+}) => {
+	const [context/*, dispatch*/] = getEnonicContext();
+
+	const properties = getIn(context.values, 'properties');
+	console.debug('properties', properties);
```
Comment (poi33) [inline src/main/resources/assets/react/documentTypes/FieldsList.jsx:29]: debug

File: src/main/resources/assets/react/documentTypes/FieldsList.jsx [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+						fulltext,
+						includeInAllText,
+						max,
+						min,
+						name,
+						nGram,
+						path,
+						valueType
+					}, index) => {
+						console.debug('nGram', nGram);
```
Comment (poi33) [inline src/main/resources/assets/react/documentTypes/FieldsList.jsx:104]: debug

File: src/main/resources/services/graphQL/graphQL.es [hunk 3]
```diff
@@ -277,12 +282,17 @@ export function post(request) {
 	};
 	//log.info(`context:${toStr(context)}`);
 
+	const before = currentTimeMillis();
+	const obj = execute(SCHEMA, query, variables, context);
+	const after = currentTimeMillis();
+	const duration = after - before;
+	log.debug(`Duration: ${duration}ms Query:${query}`);
```
Comment (poi33) [inline src/main/resources/services/graphQL/graphQL.es:289]: debug

File: src/main/resources/assets/react/documentTypes/FieldsList.jsx (hunk 2 above)
Comment (ComLock) [reply src/main/resources/assets/react/documentTypes/FieldsList.jsx:104]: Fixed in next PR

**Pull request merged**